google_nest_sdm/webrtc_util.py

```diff
@@ -1,114 +1,108 @@
-# SDP Direction Constants
-DIRECTION_SENDRECV = 'sendrecv'
-DIRECTION_SENDONLY = 'sendonly'
-DIRECTION_RECVONLY = 'recvonly'
-DIRECTION_INACTIVE = 'inactive'
+"""Library with functions for manipulating WebRTC requests/responses."""
+from enum import StrEnum
+
+# SDP direction constants
+class SDPDirection(StrEnum):
```

Comment:
You can replace comments with pydoc.

```suggestion
class SDPDirection(StrEnum):
    """SDP direction constants."""
```